Python/Wikipedia_Scraper/README.md

```diff
@@ -0,0 +1,29 @@
+# Wikipedia scraper to get a Football player's club career information  
+
+- Using BeautifulSoup, a python library useful for web scraping, this script helps to scrape Wikipedia to obtain a Football player's club career data
+
+- In this script, we take the input of the name of any popular Football player from the user and generate the club career details associated with them.
+
+## Setup Instructions
+
+### Requirements
+
+1. BeautifulSoup library:
```

Comment:
Considering you have a `requirements.txt`

Consider using `pip3 install -r requirements.txt`

in place of bs4 and request

You can list out dependencies if you want as list though